src/main/java/com/emc/ecs/nfsclient/network/Connection.java

```diff
@@ -297,7 +297,9 @@ protected void connect() throws RpcException {
         final ChannelFuture oldChannelFuture = _channelFuture;
 
         String logPrefix = _usePrivilegedPort ? "usePrivilegedPort " : "";
-        LOG.info(logPrefix + "connecting to {}", getRemoteAddress());
+        if (LOG.isDebugEnabled()) {
+            LOG.debug(logPrefix + "connecting to {}", getRemoteAddress());
```

Comment:
logPrefix makes sense, but I wasn't sure if we wanted the method call overhead. That could get expensive later, even if it's a cheap method now.